staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go

```diff
@@ -190,6 +190,14 @@ func IsValidPortNum(port int) []string {
 	return []string{InclusiveRangeError(1, 65535)}
 }
 
+// IsValidPortNumWithZeroInclusive tests that the argument is a valid port number, with zero inclusive.
+func IsValidPortNumWithZeroInclusive(port int) []string {
```

Comment:
I thought there might be some other ports which allow 0 value. I remembered that I saw a use case when I reviewed PRs. But I'm okay to remove it here.